content/guides/datomic/src/ports.datomic/peer.clj

```diff
@@ -0,0 +1,18 @@
+(ns ports.datomic.peer
+  (:require [datomic.api :as d :refer (q)]))
+
+(def uri "datomic:dev://localhost:4334/hello")
+
+(def schema-tx (read-string (slurp "resources/hello/schema.edn")))
+(def data-tx (read-string (slurp "resources/hello/seed-data.edn")))
+
+(defn init-db []
+      (println "starting db!")
+      (d/create-database uri)
+      (let [conn (d/connect uri)]
+           (println @(d/transact conn schema-tx))
```

Comment:
I'd prefer to see this as structured logging.

Reply:
I added pedestal log into it, not sure if it suits well the use case here